css/about-styles.css

```diff
@@ -0,0 +1,57 @@
+.paw-logo {
+  display: inline-block;
+}
+
+.main-content {
+  padding: 208px 92px 320px;
+}
+
+.founder-section {
+  display: flex;
+  flex-flow: row;
+}
+
+.founder-section__text {
+  margin-left: 57px;
+  text-align: center;
+}
+
+.founder-section__title {
+  width: 354px;
+  height: 43px;
+}
+
+.founder-section__p {
+  width: 357px;
+  height: 117px;
+}
+
+.founder-section__img {
+  width: 524px;
+  height: 296px;
+}
+
+.community-section {
+  display: flex;
+  flex-flow: row;
+  padding-top: 183px;
+}
+
+.community-section__title {
+  width: 500px;
+  height: 58px;
+  text-align: left;
+}
+
+.community-section__p {
+  width: 414px;
+  height: 148px;
+  text-align: left;
+  margin-right: 30px;
+}
+
+.community-section__img {
+  width: 492px;
+  height: 380px;
+  opacity: 0.8px;
```

Comment:
is opacity written without "px"? pls check this. I have a feeling the img does not have opacity rn

Reply:
Yes, you are right! Didn't realise I've put pixels there. Thank you!